src/about.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Tibeb :About</title>
+  </head>
+  <body>
+    <div class="mobile-menu display-none">
+      <img src="../assets/x-icon.png" alt="x icon" class="x-icon" />
+    </div>
+    <ul class="top-nav">
+      <li class="top-nav__facebook"><i class="fab fa-facebook-square"></i></li>
+      <li class="top-nav__twiter"><i class="fab fa-twitter-square"></i></li>
+      <li class="top-nav__English">English</li>
+      <li class="top-nav__my-page">My Page</li>
+      <li class="top-nav__logout">Logout</li>
+    </ul>
+
+    <nav>
+      <h2 class="nav__link header__nav--logo dn">
+        <a href="index.html">Tibeb <span>Bootcamp</span></a>
+      </h2>
+      <ul class="nav dn">
```

Comment:
- [ ] Kindly add a `Home` link to the about page. You already have the link to the homepage in the logo, but take into count that not all users realize of this and sometimes they will be trying to find the `Home` button, so it's always good to have one even if you already set that link to your logo.

_Note: It would be even better if you can make the same `About` button from the homepage change depending on which page you are, but this is not mandatory, just do it if you feel like doing it! :shipit:_